internal/names.go

```diff
@@ -14,6 +14,11 @@ const (
 	ImageLabel = "com.docker.application"
 )
 
+var (
+	// Application file names, in order.
+	FileNames = []string{"metadata.yml", "docker-compose.yml", "settings.yml"}
```

Comment:
Should we put those names into constants ? and that thing in a constant too :angel: (so that we can't change it :stuck_out_tongue_closed_eyes:)

Reply:
Can't put a slice as a constant sorry...

Reply:
@silvin-lubecki right, but the value inside the slice could :stuck_out_tongue_closed_eyes:

Reply:
👍